pvlib/tests/test_irradiance.py

```diff
@@ -203,7 +203,7 @@ def test_reindl(irrad_data, ephem_data, dni_et):
         40, 180, irrad_data['dhi'], irrad_data['dni'], irrad_data['ghi'],
         dni_et, ephem_data['apparent_zenith'], ephem_data['azimuth'])
     # values from matlab 1.4 code
```

Comment:
does this comment need to be updated?

Reply:
I think it's OK, can't have been meant literally since Matlab doesn't produce np.nan.